src/send/SendEnterAmount.tsx

```diff
@@ -216,10 +216,6 @@ function SendEnterAmount({ route }: Props) {
     if (parsedAmount.isLessThanOrEqualTo(0) || parsedAmount.isGreaterThan(token.balance)) {
       return
     }
-    // TODO (ACT-973?): Address is not always set on the recipient, even for PN recipients.
```

Comment:
this is no longer true, we have static typing to ensure address is always present at this point